### PR TITLE
dojson: clear forgotten dedupes.

### PR DESCRIPTION
**Pull request by mihaibivol**
Signed-off-by: Mihai Bivol <mihai.bivol@cern.ch>

### CODE REVIEW

File: tests/unit/dojson/test_dojson_institutions.py [hunk 1]
```diff
@@ -538,24 +538,6 @@ def test_hidden_notes_from_double_595__a():
     assert expected == result['hidden_notes']
 
 
-def test_hidden_notes_from_double_595__a_removes_duplicates():
```
Comment (jacquerie) [inline tests/unit/dojson/test_dojson_institutions.py]: Why was this removed?

Comment (mihaibivol) [inline tests/unit/dojson/test_dojson_institutions.py]: Because we don't remove duplicates in the rules and just in the end.

The test would have looked like:
```
result = clear_duplicates(dojson(smth))
assert no duplicates
```

It doesn't make much more sense to test deduping per specific field since the rule itself is not deduping.

Comment (jacquerie) [inline tests/unit/dojson/test_dojson_institutions.py]: Oh, you're absolutely right.

**Comment by jacquerie** [comment]
LGTM minus a terribly small nit: you have a dot at the end of the commit title : (
